bcgov/House-Innovations-HOUSNAV
# HOUSNAV-200: Updated Application State

[HOUSNAV-200](https://hous-bssb.atlassian.net/browse/HOUSNAV-200)

## Overview & Purpose
Update the application state to account for a walkthrough that has multiple sets of walkthrough questions and results.

## Summary of Changes
- Updated walkthrough URLS to use a query parameter
- Query parameter(s) are used to build the full walkthrough data
- Updated application store code to account for the possibility of multiple walkthroughs with different sets of questions and results

## Side Effects
All of them?

## Testing
- Make sure the current single dwelling walkthroughs work
- Make sure different configurations of multi dwelling walkthroughs allow the user to get to a result

## Notes & Resources
- The step tracker and results updates are captured in separate tickets

## Checklist
